monai/inferers/inferer.py

```diff
@@ -221,3 +221,68 @@ def __call__(self, inputs: torch.Tensor, network: nn.Module, *args: Any, **kwarg
             cam = GradCAMpp(network, self.target_layers, *self.args, **self.kwargs)
 
         return cam(inputs, self.class_idx, *args, **kwargs)
+
+
+class SliceInferer(SlidingWindowInferer):
+    """
+    SliceInferer extends SlidingWindowInferer to provide slice-by-slice (2D) inference
+    when provided a 3D volume.
+
+    Args:
+        spatial_dim: Spatial dimension over which the slice-by-slice inference runs on the 3D volume.
+            For example ``0`` could slide over axial slices. ``1`` over coronal slices and ``2`` over sagittal slices.
+        args: other optional args to be passed to the `__init__` of SliceInferer.
```

Comment:
I think it should be: "passed to the `__init__` of base class SlidingWindowInferer".

Thanks.